semgrep-core/tests/generate_test_matrix.py

```diff
@@ -40,6 +40,15 @@
     "exprstmt": "Expression and Statement",
 }
 
+LANGUAGE_EXCEPTIONS = {
+    "java": [
+        "naming_import",
```

Comment:
Is there an issue tracking this?

Reply:
From what I can tell, Java doesn't support import aliasing. To address the comment below as well, this list _is_ the explicit declaration that a language doesn't support something. A missing test tells me "someone forgot to test this, or maybe it's not supported" whereas this list tells me "this language doesn't support this feature." Or, at least, that's what I was hoping. Is there a better way we can state that?